echion/stacks.h

```diff
@@ -188,10 +115,8 @@ unwind_frame(PyObject *frame_addr, FrameStack &stack)
         try
         {
 #if PY_VERSION_HEX >= 0x030b0000
-            auto resolved_address = stack_chunk ? stack_chunk->resolve(current_frame_addr) : current_frame_addr;
-            Frame &frame = resolved_address != current_frame_addr
-                ? Frame::read_local((_PyInterpreterFrame *)resolved_address, &current_frame_addr)
-                : Frame::read((PyObject *)current_frame_addr, &current_frame_addr);
```

Comment:
Here we resolve the address of the current frame we are processing, and if it doesn't match the original address we do the local read from the stack chunk. The function then updates the current frame to point to the previous one, if available. Is it here that we are not resolving all the frame addresses?

Reply:
From CPython 3.13, we need to iterate the frame objects until we see `f_executable`'s type is code object. While doing that we need to resolve those frame addresses.

Reply:
> For CPython 3.13+, we need to iterate the frame objects until we see f_executable's type is code object.

Ah, of course, I haven't thought about that! Thanks for explaining!

Reply:
No problem. I'll update the code more and will ping you again.